docs/guides/utilizing-forms.md

```diff
@@ -0,0 +1,71 @@
+---
+id: utilizing-forms
+title: Building forms with Modeler
+---
+
+:::note
+The initial release of Camunda Forms includes a debut minimal feature set, which will be expanded with upcoming versions.
+
+If using with Camunda Platform, note that the Camunda Forms feature was added with the 4.7.0 release of the Camunda Modeler. Therefore, they can be used within BPMN diagrams running on Camunda Platform version 7.15.0 or later.
+:::
+
+# Overview
+
+The Camunda Forms feature allows you to easily design and configure forms. Once configured, they can be connected to a user task or start event to implement a task form in your application.
+
+# Quickstart
+
+## Create new form
+
+To start building a form, log in to your [Camunda Cloud](./getting-started/create-camunda-cloud-account.md) account or open [Camunda Modeler](./components/modeler/what-is-camunda-modeler.md) and take the following steps:
+
+1. Click on the **Forms** tab at the top of the page or open the **File** menu in Modeler.
+2. Click the blue **Create New Form** button.
+
+## Build your form
+
+Now you can start to build your Camunda form. First, let's name our form by clicking the three vertical dots next to the text reading **New Form** in the top left corner of the page. Select **Rename** to give your form a new name. In this example, we'll analyze a form to help with a task in obtaining an email message.
```

Comment:
What do you refer to with _analyze_ here?

Reply:
Adjusted to **build** here, in latest commit.